react/src/theme/components/Menu.ts

```diff
@@ -46,6 +46,7 @@ const baseStyle = definePartsStyle((props) => {
       boxShadow: $shadow.reference,
     },
     item: {
+      mt: '0.125rem',
```

Comment:
oki! can i just double check why we wanna put this into `sizes` rather than `baseStyles`? (no opinion, cos idk if got diff sizes from the figma + idk if the border will change b/w sizes)

Reply:
future proofing, since sizes control the things that may change between sizes.

Reply:
yea i think i'll put this in `variant` and define a `MENU_ITEM_BORDER_STYLES` to share b/w `outline` + `clear` (i could define a function also but cos we define `clear` in terms of a function so idk if it would be confusing)